tests/Test.Yaapii.Xml/XMLSliceTests.cs

```diff
@@ -104,5 +106,57 @@ public void FileCtorAppliesEncoding(string name)
                 );
             }
         }
+
+        [Fact]
+        public void ValuesKnowsNamespace()

```

Comment:
these tests here also

Reply:
I think values method does not "know" a namespace, since it is not an object. It is a procedure of an object, so it may apply or work with a namespace.